xdsl/builder.py

```diff
@@ -55,22 +55,34 @@ class Builder(BuilderListener):
     """Operations will be inserted at this location."""
 
     def insert(self, op: OperationInvT) -> OperationInvT:
```

Comment:
Should we deprecated this? There are 76 uses in xDSL, and most don't use the result, or can trivially be refactored.

Reply:
I think we should just merge it with `insert_op`, which should be the same method anyway!